package.json

```diff
@@ -298,7 +298,8 @@
 		"svgstore-cli": "^1.3.1",
 		"terser": "^5.5.1",
 		"terser-webpack-plugin": "^5.1.1",
-		"typescript": "^4.1.3",
+		"tslib": "^2.1.0",
```

Comment:
Out of curiosity, why did we not have to put `tslib` here before?

Reply:
`tslib@1.11.2` got hoisted from one of `packages/.../node_modules/tslib` into `node_modules`.

This PR ensures we have an explicit dependency on `tslib` on the root, which is where we run `tsc` from.